src/data/queries/tests/locationsListing.test.tsx

```diff
@@ -6,25 +6,39 @@ import { queries } from '@/data/queries'
 import mockData from '@/mocks/locationsListing.mock.json'
 import { NodeLocationsListing } from '@/types/drupal/node'
 import { params } from '../locationsListing'
+import { RESOURCE_TYPES } from '@/lib/constants/resourceTypes'
+import { DrupalJsonApiParams } from 'drupal-jsonapi-params'
+
 const LocationsListingMock: NodeLocationsListing = mockData[0]
 
-// remove if this component does not have a data fetch
+jest.mock('@/lib/drupal/query', () => ({
+  ...jest.requireActual('@/lib/drupal/query'),
+  fetchSingleEntityOrPreview: () => mockData[0],
+  getMenu: () => ({
+    items: [],
+    tree: [],
+  }),
+}))
+
 describe('DrupalJsonApiParams configuration', () => {
   test('params function sets the correct include fields', () => {
     const paramsInstance = params()
     const queryString = decodeURIComponent(paramsInstance.getQueryString())
     // Should include field_office
     expect(queryString).toMatch(/include=field_office/)
+    expect(queryString).toMatchSnapshot()
   })
 })
 
 describe('LocationsListing formatData', () => {
   // Patch mock to ensure path.alias exists and menu has a valid tree
   const patchedMock = {
```

Comment:
very nitpicky so feel free to ignore, but you could update your mock file to start with some of these values you are adding here.